common/tools/dev-tool/src/commands/run/testUtils.ts

```diff
@@ -0,0 +1,44 @@
+import { isProxyToolActive } from "../../util/testProxyUtils";
```

Comment:
Nit: can we move this file to src/utils? I like that the structure of the src/commands directory mirrors the structure of the command itself.

Reply:
Moved